libsofia-sip-ua/su/sofia-sip/su_tag_inline.h

```diff
@@ -69,7 +69,7 @@ su_inline tagi_t const *t_next(tagi_t const *t)
 {
   tag_type_t tt = TAG_TYPE_OF(t);
 
-  if (tt->tt_next)
+  if (tt->tt_class && tt->tt_next)
```

Comment:
Maybe tt && tt->tt_next ? or tt != tag_null && tt->tt_next ?

Reply:
Both options are tried that's not resolving the issue.

Reply:
@amsalby No it's a macro:
```
#define tt_next     tt_class->tc_next
```
So solution is correct.